Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM centos:6
```

Comment:
Good point, @gtorodelvalle. I have the same concerns. It's mostly a style choice so, as far as it works with the automated builds, I really don't mind which approach we take.